build/scripts/generate_devworkspace_templates.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Copyright (c) 2019-2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+mkdir /build/out/
+for dir in /build/devfiles/*/
+do
+  devfile=$(cat ${dir}meta.yaml | yq -r '."links"."v2"')
+  if [ $devfile != null ]; then
+    dir=${dir%/}
+    dir=/build/out/${dir##*/}
+    mkdir ${dir}
+    npx @eclipse-che/che-theia-devworkspace-handler --devfile-url:$devfile --output-file:${dir}/meta.yaml
```

Comment:
the output should be with a different name than meta.yaml as we're generating a file that is containing a DevWorkspace + DevWorkspaceTemplate

Reply:
it looks overkill to install jq and yq just to know if there is v2.links in the meta.yaml while probably only a grep expression is enough

Reply:
done

Reply:
>the output should be with a different name than meta.yaml as we're generating a file that is containing a DevWorkspace + DevWorkspaceTemplate

@benoitf can we just name it `devWorkspace.yaml`?

Reply:
yes why not it's better than meta.yaml

Reply:
done